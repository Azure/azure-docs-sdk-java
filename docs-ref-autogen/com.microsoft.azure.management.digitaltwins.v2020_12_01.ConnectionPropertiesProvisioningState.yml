### YamlMime:JavaType
uid: "com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState"
fullName: "com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState"
name: "ConnectionPropertiesProvisioningState"
nameWithType: "ConnectionPropertiesProvisioningState"
summary: "Defines values for Connection<wbr>Properties<wbr>Provisioning<wbr>State."
inheritances:
- "<xref href=\"java.lang.Object?displayProperty=fullName\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.microsoft.rest.ExpandableStringEnum?displayProperty=fullName\" data-throw-if-not-resolved=\"False\" />"
inheritedMembers:
- "com.microsoft.rest.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)"
- "com.microsoft.rest.ExpandableStringEnum.<T>values(java.lang.Class<T>)"
- "com.microsoft.rest.ExpandableStringEnum.equals(java.lang.Object)"
- "com.microsoft.rest.ExpandableStringEnum.hashCode()"
- "com.microsoft.rest.ExpandableStringEnum.toString()"
- "com.microsoft.rest.ExpandableStringEnum.withNameValue(java.lang.String,T,java.lang.Class<T>)"
- "java.lang.Object.clone()"
- "java.lang.Object.finalize()"
- "java.lang.Object.getClass()"
- "java.lang.Object.notify()"
- "java.lang.Object.notifyAll()"
- "java.lang.Object.wait()"
- "java.lang.Object.wait(long)"
- "java.lang.Object.wait(long,int)"
syntax: "public final class ConnectionPropertiesProvisioningState extends ExpandableStringEnum<ConnectionPropertiesProvisioningState>"
constructors:
- uid: "com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState.ConnectionPropertiesProvisioningState()"
  fullName: "com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState.ConnectionPropertiesProvisioningState()"
  name: "ConnectionPropertiesProvisioningState()"
  nameWithType: "ConnectionPropertiesProvisioningState.ConnectionPropertiesProvisioningState()"
  syntax: "public ConnectionPropertiesProvisioningState()"
fields:
- uid: "com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState.APPROVED"
  fullName: "com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState.APPROVED"
  name: "APPROVED"
  nameWithType: "ConnectionPropertiesProvisioningState.APPROVED"
  summary: "Static value Approved for Connection<wbr>Properties<wbr>Provisioning<wbr>State."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState?alt=com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState&text=ConnectionPropertiesProvisioningState\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final ConnectionPropertiesProvisioningState APPROVED"
  desc: "Static value Approved for ConnectionPropertiesProvisioningState."
- uid: "com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState.DISCONNECTED"
  fullName: "com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState.DISCONNECTED"
  name: "DISCONNECTED"
  nameWithType: "ConnectionPropertiesProvisioningState.DISCONNECTED"
  summary: "Static value Disconnected for Connection<wbr>Properties<wbr>Provisioning<wbr>State."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState?alt=com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState&text=ConnectionPropertiesProvisioningState\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final ConnectionPropertiesProvisioningState DISCONNECTED"
  desc: "Static value Disconnected for ConnectionPropertiesProvisioningState."
- uid: "com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState.PENDING"
  fullName: "com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState.PENDING"
  name: "PENDING"
  nameWithType: "ConnectionPropertiesProvisioningState.PENDING"
  summary: "Static value Pending for Connection<wbr>Properties<wbr>Provisioning<wbr>State."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState?alt=com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState&text=ConnectionPropertiesProvisioningState\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final ConnectionPropertiesProvisioningState PENDING"
  desc: "Static value Pending for ConnectionPropertiesProvisioningState."
- uid: "com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState.REJECTED"
  fullName: "com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState.REJECTED"
  name: "REJECTED"
  nameWithType: "ConnectionPropertiesProvisioningState.REJECTED"
  summary: "Static value Rejected for Connection<wbr>Properties<wbr>Provisioning<wbr>State."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState?alt=com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState&text=ConnectionPropertiesProvisioningState\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final ConnectionPropertiesProvisioningState REJECTED"
  desc: "Static value Rejected for ConnectionPropertiesProvisioningState."
methods:
- uid: "com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState.fromString(java.lang.String)"
  fullName: "com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState.fromString(String name)"
  name: "fromString(String name)"
  nameWithType: "ConnectionPropertiesProvisioningState.fromString(String name)"
  summary: "Creates or finds a Connection<wbr>Properties<wbr>Provisioning<wbr>State from its string representation."
  modifiers:
  - "static"
  parameters:
  - description: "a name to look for"
    name: "name"
    type: "<xref href=\"java.lang.String?alt=java.lang.String&text=String\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static ConnectionPropertiesProvisioningState fromString(String name)"
  desc: "Creates or finds a ConnectionPropertiesProvisioningState from its string representation."
  returns:
    description: "the corresponding ConnectionPropertiesProvisioningState"
    type: "<xref href=\"com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState?alt=com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState&text=ConnectionPropertiesProvisioningState\" data-throw-if-not-resolved=\"False\" />"
- uid: "com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState.values()"
  fullName: "com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState.values()"
  name: "values()"
  nameWithType: "ConnectionPropertiesProvisioningState.values()"
  modifiers:
  - "static"
  syntax: "public static Collection<ConnectionPropertiesProvisioningState> values()"
  returns:
    description: "known ConnectionPropertiesProvisioningState values"
    type: "<xref href=\"java.util.Collection?alt=java.util.Collection&text=Collection\" data-throw-if-not-resolved=\"False\" />&lt;<xref href=\"com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState?alt=com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState&text=ConnectionPropertiesProvisioningState\" data-throw-if-not-resolved=\"False\" />&gt;"
type: "class"
desc: "Defines values for ConnectionPropertiesProvisioningState."
metadata: {}
package: "com.microsoft.azure.management.digitaltwins.v2020_12_01"
artifact: com.microsoft.azure.digitaltwins.v2020_12_01:azure-mgmt-digitaltwins:1.0.0
