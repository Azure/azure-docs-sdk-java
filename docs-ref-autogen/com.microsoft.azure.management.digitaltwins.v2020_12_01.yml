### YamlMime:JavaPackage
uid: "com.microsoft.azure.management.digitaltwins.v2020_12_01"
fullName: "com.microsoft.azure.management.digitaltwins.v2020_12_01"
name: "com.microsoft.azure.management.digitaltwins.v2020_12_01"
summary: "This package contains the classes for AzureDigitalTwinsManagementClient.\n Azure Digital Twins Client for managing DigitalTwinsInstance."
classes:
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.AuthenticationType"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.CheckNameRequest"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionProperties"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesPrivateEndpoint"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesPrivateLinkServiceConnectionState"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionState"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsEndpointResourceProperties"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsIdentity"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsIdentityType"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsPatchDescription"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsPatchProperties"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsResource"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.EndpointProvisioningState"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.ErrorDefinition"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.ErrorResponse"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.ErrorResponseException"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.EventGrid"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.EventHub"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.ExternalResource"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.GroupIdInformationProperties"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.GroupIdInformationPropertiesModel"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.OperationDisplay"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateEndpoint"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateEndpointConnectionProperties"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateLinkServiceConnectionStatus"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.ProvisioningState"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PublicNetworkAccess"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.Reason"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.ServiceBus"
interfaces:
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.CheckNameResult"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwins"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsDescription"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsDescription.Definition"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsDescription.DefinitionStages"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsDescription.DefinitionStages.Blank"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsDescription.DefinitionStages.WithCreate"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsDescription.DefinitionStages.WithGroup"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsDescription.DefinitionStages.WithIdentity"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsDescription.DefinitionStages.WithPrivateEndpointConnections"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsDescription.DefinitionStages.WithPublicNetworkAccess"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsDescription.Update"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsDescription.UpdateStages"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsDescription.UpdateStages.WithIdentity"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsDescription.UpdateStages.WithProperties"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsEndpointResource"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsEndpointResource.Definition"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsEndpointResource.DefinitionStages"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsEndpointResource.DefinitionStages.Blank"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsEndpointResource.DefinitionStages.WithCreate"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsEndpointResource.DefinitionStages.WithDigitalTwinsInstance"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsEndpointResource.DefinitionStages.WithProperties"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsEndpointResource.Update"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsEndpointResource.UpdateStages"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsEndpointResource.UpdateStages.WithProperties"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsEndpoints"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.GroupIdInformation"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.GroupIdInformationResponse"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.Operation"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.Operations"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateEndpointConnection"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateEndpointConnection.Definition"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateEndpointConnection.DefinitionStages"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateEndpointConnection.DefinitionStages.Blank"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateEndpointConnection.DefinitionStages.WithCreate"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateEndpointConnection.DefinitionStages.WithDigitalTwinsInstance"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateEndpointConnection.DefinitionStages.WithProperties"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateEndpointConnection.Update"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateEndpointConnection.UpdateStages"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateEndpointConnection.UpdateStages.WithProperties"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateEndpointConnections"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateEndpointConnectionsResponse"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateLinkResources"
metadata: {}
package: "com.microsoft.azure.management.digitaltwins.v2020_12_01"
artifact: com.microsoft.azure.digitaltwins.v2020_12_01:azure-mgmt-digitaltwins:1.0.0
