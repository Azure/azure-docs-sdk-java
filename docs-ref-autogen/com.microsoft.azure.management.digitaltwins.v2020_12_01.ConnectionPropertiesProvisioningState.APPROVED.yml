### YamlMime:JavaMember
uid: "com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState.APPROVED*"
fullName: "com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState.APPROVED"
name: "APPROVED"
nameWithType: "ConnectionPropertiesProvisioningState.APPROVED"
members:
- uid: "com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState.APPROVED"
  fullName: "com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState.APPROVED"
  name: "APPROVED"
  nameWithType: "ConnectionPropertiesProvisioningState.APPROVED"
  summary: "Static value Approved for ConnectionPropertiesProvisioningState."
  syntax: "public static final ConnectionPropertiesProvisioningState APPROVED"
type: "field"
metadata: {}
package: "com.microsoft.azure.management.digitaltwins.v2020_12_01"
artifact: com.microsoft.azure.digitaltwins.v2020_12_01:azure-mgmt-digitaltwins:1.0.0
